nemo_curator/__init__.py

```diff
@@ -16,11 +16,17 @@
 
 # Disable query planning if possible
 # https://github.com/NVIDIA/NeMo-Curator/issues/73
-if dask.config.get("dataframe.query-planning") is True:
+QUERY_PLANNING = dask.config.get("dataframe.query-planning")
+if QUERY_PLANNING is True or QUERY_PLANNING is None:
```

Comment:
Thinking a bit more about this, it might make sense to replace the `None` check for the config with something like `"dask_expr" in sys.modules`. 
We can then followup with a better approach based on the resolution of upstream dask discussions.